climada_petals/entity/exposures/gdp_asset.py

```diff
@@ -240,7 +240,7 @@ def _fast_impf_mapping(countryID, natID_info):
         float,float
         """
     nat = natID_info['ID']
-    impf_RF = natID_info['impf_RF']
+    impf_RF = natID_info['if_RF']
```

Comment:
Could you try to remove the NatRegIDs.csv file from your climada/data folder and run the code again? This should download the file again, I think that you may have an older version where if was not updated to impf yet. It seems like the column is named 'impf_RF' in the repo version https://github.com/CLIMADA-project/climada_python/blob/main/data/system/NatRegIDs.csv .

Reply:
Right, converted this back in 6aa44ca